adoc-mode.el

```diff
@@ -552,23 +921,35 @@ To become a customizable variable when regexps for list items become customizabl
 (defun adoc-re-attribute-entry ()
   (concat "^\\(:[a-zA-Z0-9_][^.\n]*?\\(?:\\..*?\\)?:[ \t]*\\)\\(.*?\\)$"))
 
+(defun adoc-update-header-faces (&optional scaling scaling-values)
+  "Update header faces, depending on if header SCALING is desired.
+If so, use given list of SCALING-VALUES relative to the baseline
+size of `markdown-header-face'."
+  (dotimes (num 6)
+    (let* ((face-name (intern (format "adoc-header-face-%s" (1+ num))))
+           (scale (cond ((not scaling) 1.0)
+                        (scaling-values (float (nth num scaling-values)))
+                        (t (float (nth num adoc-header-scaling-values))))))
+      (unless (get face-name 'saved-face) ; Don't update customized faces
+        (set-face-attribute face-name nil :height scale)))))
+
 ;; from asciidoc.conf: ^= +(?P<title>[\S].*?)( +=)?$
 ;; asciidoc src code: Title.isnext reads two lines, which are then parsed by
 ;; Title.parse. The second line is only for the underline of two line titles.
 (defun adoc-re-one-line-title (level)
   "Returns a regex matching a one line title of the given LEVEL.
-When LEVEL is nil, a one line title of any level is matched.
-
-match-data has these sub groups:
-1 leading delimiter inclusive whites between delimiter and title text
-2 title's text exclusive leading/trailing whites
-3 trailing delimiter with all whites
-4 trailing delimiter only inclusive whites between title text and delimiter
-0 only chars that belong to the title block element
-
-==  my title  ==  n
----12------23------
-            4--4"
+  When LEVEL is nil, a one line title of any level is matched.
```

Comment:
In Elisp the current indentation level was actually the right one, so please undo this type of changes.

Reply:
Hello! Fixed.